### ISSUE TITLE
scrollIntoView() does not work

### ISSUE THREAD

**Issue by ddeath** [cypress-io/cypress#519]
* Operating System: Linux Mint 18
* Cypress Version: 0.19.2
* Browser/Browser Version: Chrome 60, Electron 53

### Are you requesting a feature or reporting a bug?
Bug

### Current behavior:
Test are failing because the code does not scroll into the element

### Expected behavior:
Test are passing because it scrolls into the element

### How to reproduce the current behavior:
clone [`https://github.com/ddeath/cypress-scroll-bug`](https://github.com/ddeath/cypress-scroll-bug) and run the tests

#### Test code:

```js
describe('Broken Sink', function(){
  before(() => {
    cy.visit('/');
  });

  it('Should scroll in cli', () => {
    cy.get('.selector').click();
    cy.get('.item-2').click();

    cy.get('#footer').then(function($btn){
      $btn.get(0).scrollIntoView();
    }).get('.selector').click()
      .get('.item-3').click();
  });
})
```

**Comment by RandallKent** [comment]
The code for this is done, but this has yet to be released. We'll update the issue and reference the changelog when it's released.

**Comment by brian-mann** [comment]
Fixed in [`0.20.0`](https://docs.cypress.io/guides/references/changelog.html#0-20-0)
